src/Chart.PieceLabel.js

```diff
@@ -95,11 +95,14 @@
         innerRadius = view.innerRadius;
         position = element.tooltipPosition();
       }
-
+      
+	  var fontColor = typeof this.fontColor === 'string' 
+				? this.fontColor 
+				: this.fontColor.length > i ? this.fontColor[i] : this.fontColor[this.fontColor.length - 1];
```

Comment:
behavior of backgroundColor is fallback to the default color.
```JavaScript
{
  data: [50445, 33655, 15900],
  backgroundColor: ['#FF6384', '#36A2EB']
}
```
third color will be default gray, so we can follow this. Current version default font color is `#fff`. But I also consider changing to `options.defaultFontColor`